README.md

```diff
@@ -129,6 +129,18 @@ An example of setting the environment variable `VST2SDK_DIR` would be:
 
 ***NOTE***: This environment variable needs to be set _before_ running `premake5 xcode4` - which generates projects / and is part of the `build-osx.sh` script.
 
+## Building a Surge.vst3 (VST3) with OSX/macOS
```

Comment:
I have no trouble building/packaging/running VST3, I think the reason that was added is rendered moot by this PR.

Reply:
OK wanna just fix it in place in this PR? You can edit the README directly and I'll approve it. Thanks for the careful eyes!

And does vst3 work for you? If so I can look at zoom this week!!

Reply:
Actually I've just done a little more testing, it's a bit odd:

Live10 : VST2 works, AU won't load at all (VST3 not supported).
Reaper : VST3 works, AU loads in some weird bridge mode and I don't get sound, VST2 doesn't show up at all.
Cubase 9.5 : VST2 works, VST3 won't load at all (AU not supported).
Cubase 8.5 : VST3 loads but doesn't produce sound, no sign of VST2.
Logic Pro X 10.4.1 : AU works (VST of course not supported)

Reply:
sure, no worries. it's just that i haven't seen anyone dump a .vst3 and go "well this definitely works". if it just works right off the bat (clone repo, submodules, build-osx, xcode and package) then that's great, just add it in as text :)

Reply:
OK well that sounds roughly like it could be described as "here be dragons" for now. Why don't I merge this so we can close this thread then if/when we get a vst working across a couple of places we can update the build instructions with that sweep.

Sorry this took so long!